pkg/api/soc.go

```diff
@@ -155,6 +157,15 @@ func (s *Service) socUploadHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	encryptedReference := sch.Address()
+	if headers.Act {
+		encryptedReference, err = s.actEncryptionHandler(r.Context(), w, putter, sch.Address(), headers.HistoryAddress)
```

Comment:
Let's use the assigned variable instead of `sch.Address()`

Reply:
call it reference and reuse it as argument of `s.actEncryptionHandler`

Reply:
renamed to reference